manifests/dev/vsphere-7.0u2/vanilla/rbac/vsphere-csi-node-rbac.yaml

```diff
@@ -14,7 +14,7 @@ rules:
     resources: ["configmaps"]
     verbs: ["get", "list", "watch"]
 ---
-kind: ClusterRoleBinding
+kind: RoleBinding
 apiVersion: rbac.authorization.k8s.io/v1
 metadata:
   name: vsphere-csi-node-cluster-binding
```

Comment:
Same comment as above

Reply:
After merging the rules, looks like `vsphere-csi-node-cluster-role` can be removed. Take a look at the latest commit